lib/python/picongpu/input/parameters.py

```diff
@@ -166,7 +199,8 @@ def __init__(self, name, ptype, unit, default, slider_min, slider_max,
             internal value.
         """
         UiParameter.__init__(self, name, ptype, unit, default,
-                             slider_min, slider_max, slider_step, value)
+                             slider_min, slider_max, slider_step,
+                             label, formatter, value, dtype)
```

Comment:
for some reason when executing
```
p = LinearScaledParameter(
         name="Wave_Length_SI", ptype="compile", unit="nm",
         default=800.0, slider_min=400.0, slider_max=1400.0,
         slider_step=1, scale_factor=1.e-9)
p.formatter
```        
I am getting an `AttributeError: 'LinearScaledParameter' object has no attribute 'formatter'` error. Strange since I thought this should be inherited from 'UiParameter' or 'Parameter' class ...